src/stores/useHoloStore.js

```diff
@@ -20,7 +20,7 @@ const makeUseHoloStore = ({ connectionArgs, MockWebSdk }) => defineStore('holo',
     isAvailable: state => state.agentState.isAvailable,
     isLoggedIn: state => state.agentState.isAnonymous === false && state.agentState.isAvailable === true,
     error: state => !state.agentState.isAvailable && (state.connectionError || state.agentState.unrecoverableError),
-    agentKey: (state) => state.appInfo?.cell_data[0],
+    agentKey: (state) => state.appInfo?.cell_data?.find(c => c.role_id === 'holofuel')?.cell_id[1],
```

Comment:
Oops, my bad.

But we shouldn't assume there is a `'holofuel'` `role_id`, as this is supposed to be a fully general store. We can just use any `cell_id`, as all the `cell_ids` should have the same agent key. (a `CellId` is  [DnaHash, AgentKey], so each one has it's own dna hash, but all the `cell_ids` in a given `app_info` will have the same agent key)

```suggestion
    agentKey: (state) => state.appInfo?.cell_data?.[0].cell_id[1],
```